packages/cozy-dataproxy-lib/src/search/SearchEngine.ts

```diff
@@ -359,8 +360,25 @@ export class SearchEngine {
       const FLEXSEARCH_LIMIT = 10000
       const indexResults = index.index.search(query, FLEXSEARCH_LIMIT, {
         limit: FLEXSEARCH_LIMIT,
-        enrich: true
+        enrich: false
       })
+      /*
+        Search result example:
+        [
+          {
+              "field": "displayName",
+              "result": [
+                  "604627c6bafee013ec5f27f7f72029f6"
+              ]
+          },
+          {
+              "field": "fullname",
+              "result": [
+                  "604627c6bafee013ec5f27f7f72029f6", "604627c6bafee013ec5f27f3f714568"
+              ]
+          }
+        ]
+      */
```

Comment:
Was that comment expected to be commited?

Reply:
Yes it was, it does not look like a good practice, but I honestly felt like it was helpful to better grasp the expected result quickly